packages/core/src/components/graphs/area.ts

```diff
@@ -37,18 +43,51 @@ export class Area extends Component {
 				.x((d, i) => cartesianScales.getDomainValue(d, i))
 				.y0(cartesianScales.getRangeValue(0))
 				.y1((d, i) => cartesianScales.getRangeValue(d, i));
+			minAndMax = this.services.cartesianScales.getMainYScale().domain();
```

Comment:
If you're setting `minAndMax` to a domain, shouldn't `minAndMax` just be called `domain`?